docs/source/pbeam.rst

```diff
@@ -202,8 +202,8 @@ The ``AvgStreamingOp`` provides real-time data streaming capabilities for extern
  and then streams data via ZMQ (ZeroMQ) PUB socket for efficient real-time transmission
 
 **Streaming Configuration:**
- * **Default Address:** 127.0.0.1 (localhost)
- * **Default Port:** 9798
+ * **Default Address:** 127.0.0.1 (localhost) (defined by the ``--streaming-address`` command line argument)
+ * **Default Port:** 30000 (defined by the ``--streaming-port`` command line argument), should follow a convention of dr[#] using port number 30000 + #, where # is the power beam number.
```

Comment:
I think this is too prescriptive.  If you particular behavior that should go into the service generator script so that enforced.